autoload/OmniSharp/locations.vim

```diff
@@ -55,17 +55,61 @@ function! OmniSharp#locations#Preview(location) abort
   endif
 endfunction
 
-function! OmniSharp#locations#SetQuickfix(list, title)
+function! OmniSharp#locations#SetQuickfix(list, title) abort
+  call s:SetQuickfixFromDict(a:list, {'title': a:title})
+endfunction
+
+function! OmniSharp#locations#SetQuickfixWithVerticalAlign(list, title) abort
+  call s:SetQuickfixFromDict(a:list, {'title': a:title, 'quickfixtextfunc': 's:QuickfixVerticalAlignFunction'})
+endfunction
+
+function! s:SetQuickfixFromDict(list, dict) abort
   if !has('patch-8.0.0657')
-  \ || setqflist([], ' ', {'nr': '$', 'items': a:list, 'title': a:title}) == -1
-    call setqflist(a:list)
+  \ || setqflist([], ' ', {'nr': '$', 'items': a:list, 'title': get(a:dict, 'title', 'OmniSharp')}) == -1
+    call setqflist(a:list, ' ', a:dict)
```

Comment:
This call is a fallback for older versions which cannot deal with advanced `what` arguments. It should remain "simple" (`call setqflist(a:list)`)

Reply:
Oh. It was late. I should maybe eat an apple, drink lots of coffee and get at least 4 hours of sleep starting today...